lib/src/pubspec.dart

```diff
@@ -22,8 +22,19 @@ class Pubspec {
   final Version version;
 
   final String description;
+
+  /// This should be a URL pointing to the website for the package.
   final String homepage;
 
+  /// Optional field to specify the source code repository of the package.
+  /// Useful when a package has both a home page and a repository.
+  final String repository;
+
+  /// Optional field to a web page where developers can report new issues or
+  /// view existing ones.
+  @JsonKey(name: 'issue_tracker')
```

Comment:
I just forced you to rebase here. Upside: you can drop these annotations. They are now globally configured.

Reply:
Oh, `field_rename: snake`, got it.